nginx/conf.d/1_webapp.conf

```diff
@@ -0,0 +1,47 @@
+#Modified from: https://itnext.io/docker-rails-puma-nginx-postgres-999cd8866b18
+
+upstream safecastapi {
+   server app:3000;
+}
+
+limit_req_zone $binary_remote_addr zone=one:10m rate=10r/s;
```

Comment:
This looks like a good start, but, if it is possible, it would be best if we could add this configuration to a new file instead of modifying an existing one.